src/js/strategies/CitrixVDIStrategy.js

```diff
@@ -6,28 +6,41 @@ import CCPInitiationStrategyInterface from "./CCPInitiationStrategyInterface";
 import {hitch} from "../utils";
 import {FailedState} from "../rtc_session";
 import {RTC_ERRORS} from "../rtc_const";
-import "@citrix/ucsdk/CitrixWebRTC";
+
+
 
 export default class CitrixVDIStrategy extends CCPInitiationStrategyInterface {
 
-    constructor() {
+    constructor(useRealCitrix = true) {
         super();
-        if (!window.CitrixWebRTC.isFeatureOn("webrtc1.0")) {
-            throw new Error('Citrix WebRTC redirection feature is NOT supported!');
-        }
-        window.getCitrixWebrtcRedir = function () {
-            const registryValue = Promise.resolve(1);
-            return new Promise(function (resolve, reject) {
-                //retrieve registry value internally
-                registryValue.then((v) => {
-                    resolve(v);
-                }).catch(() => {
-                    reject();
-                })
-            })
+        if(useRealCitrix){
+            require("@citrix/ucsdk/CitrixWebRTC");
         }
+        console.log("CitrixVDIStrategy initializing");
+        this.initCitrixWebRTC();
+        this.initGetCitrixWebrtcRedir();
```

Comment:
Should this be part of initCitrixWebRTC()?

Reply:
Any reason we cannot club all these separate functions in a single init function?  Seems like they are all related

Reply:
no specific reason or preference, but trying to make it more readable